slow_odgi/examples.md

```diff
@@ -0,0 +1,157 @@
+# Pollen language walkthrough
+
+## Core types in Pollen
```

Comment:
One thing I don't quite understand: is this meant to be valid Pollen source code? (And if so, is it supposed to be part of a "preamble" that will be prepended to every program?) Or is it just pseudocode, not actually meant to be parsed, but just to give humans an idea of what types are available to them?

The latter is straightforward enough, but if it's the former, there are some questions here about what these declarations are supposed to convey to the compiler. Do they describe how to load in the data, for example? If this is just meant as a human hint, maybe a brief note to that effect here would be good.

Reply:
I meant it as the latter! I'll make a note of that, and it'll live apart from the `.pol` test cases, so that it's very clear what it is.